packages/jellyfish-api-core/__tests__/category/blockchain.test.ts

```diff
@@ -249,6 +249,27 @@ describe('masternode', () => {
     })
   })
 
+  // getDifficulty of masternode & non masternode
+  describe('getdifficulty', () => {
+    // const container = new DifficultyRegTest()
+    const client = new ContainerAdapterClient(container)
+
+    beforeAll(async () => {
+      await container.start()
+      await container.waitForReady()
+      // await container.waitForWalletCoinbaseMaturity()
+    })
+
+    afterAll(async () => {
+      await container.stop()
+    })
+
```

Comment:
Remove this entire code section, it's not required. it's done by the parent `describe`.